### PR TITLE
feat(appengine): Port StaticContent sample to .NET 6

### PR DESCRIPTION
**Pull request by jahjahbin**
Same idea as #2164 and #2172 . This one seems way simpler.

### CODE REVIEW

**Review by jinglundong — APPROVED** [review]

File: appengine/flexible/StaticContent/StaticContent.Sample/Program.cs [hunk 1]
```diff
@@ -14,22 +14,16 @@
...
-    public class Program
-    {
-        public static void Main(string[] args)
-        {
-            var host = new WebHostBuilder()
-                .UseKestrel()
-                .UseContentRoot(Directory.GetCurrentDirectory())
-                .UseStartup<Startup>()
-                .Build();
+// [START gae_flex_dotnet_static_files]
```
Comment (jinglundong) [inline appengine/flexible/StaticContent/StaticContent.Sample/Program.cs:20]: I was wondering if it's safe to move this tag. From my reading of https://source.corp.google.com/search?q=gae_flex_dotnet_static_files, this should be fine.

File: appengine/flexible/StaticContent/README.md [hunk 2]
```diff
@@ -3,53 +3,7 @@
 This sample application demonstrates how to serve static content with
 Google App Engine Flexible Environment.
 
-## Prerequisites
+For detailed steps to deploy this app, check out
+[the Google Cloud
+documentation.](https://cloud.google.com/appengine/docs/flexible/serving-static-files?tab=.net)
```
Comment (jinglundong) [inline appengine/flexible/StaticContent/README.md:8]: nit: [the Google Cloud documentation.](https://...) -> [the Google Cloud
documentation](https://...).

**Review by jinglundong — APPROVED** [review]

File: appengine/flexible/StaticContent/StaticContent.Sample/Program.cs (hunk 1 above)
Comment (amanda-tarafa) [reply appengine/flexible/StaticContent/StaticContent.Sample/Program.cs:20]: If you move a tag that's being used Snippet Bot will complain and the Snippet Bot Check will be red, meaning the PR cannot be merged. On this PR Snippet Bot [is not complaining](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/2182#issuecomment-1540793623), it's just mentioning that we are removing two tags and adding two tags. So yes, this is safe.

**Review by amanda-tarafa — COMMENTED** [review]
Looks good except for the conflicts in the solution file.

Maybe accumulate the changes to the solution file in a separate PR? Else, we'll have this for all reamining PRs.

**Review by amanda-tarafa — APPROVED** [review]

**Comment by amanda-tarafa** [comment]
It's till showing as having conflicts. Can you rebase your PR locally on HEAD and fix the conflicts.

**Comment by amanda-tarafa** [comment]
Nope, that's fine, I think something like this has happened before, that the message saying "There are conflicts" is not refreshed for some reviewers even when the conflicts are fixed. I couldn't see any specific conflicts either, just a message saying there were some.

If I see this again I'll report it as an issue to GitHub.

**Pull request merged**